backend/frontend/templates/frontend/course_component_details/page.html

```diff
@@ -1,15 +1,72 @@
 {% extends "frontend/base_logged_in.html" %}
 
 {% block content %}
-    <div class="mx-auto p-6 bg-white my-4 sm:w-11/12 lg:w-8/12">
+    <div class="mx-auto p-6 bg-white my-4 sm:w-11/12 lg:w-8/12 w-full overflow-auto">
         <h1 class="{{ styles.heading1 }}">
-            {{course_component.title}}
+            {{course_component.content_item.content_type_nice | title}}: {{ course_component.content_item.title }}
         </h1>
 
-        {% if course_component.submission_type_nice == "link"%}
-            <div class="grid gap-4 my-4">
+        {% if course_component.content_item.content_type_nice == "topic"%}
+            <div class="grid gap-4 my-4 w-full">
                 <div class="p-2 border-2 rounded-md grid gap-2">
-                    <div class="bold p-2 border-2 rounded-md ">
+                    <div class="p-2 border-2 rounded-md ">
+                        Tag names: 
```

Comment:
Some shared fields like tags, status and assignees,etc. seem to be repeated. Is there no way to only write them once?